skype/skype-ps/skype/Get-CsOnlineVoiceUser.md

```diff
@@ -23,6 +23,19 @@ Get-CsOnlineVoiceUser [-CivicAddressId <XdsCivicAddressId>] [-DomainController <
 ```
 
 ## DESCRIPTION
+**NOTE:**
+
+Beginning Teams PowerShell Module 2.6.2 onwards, the below updates are applicable for TeamsOnly customers.
+
+*Deprecated Attributes*
+
+- Below attributes have been deprecated under "-ExpandLocation" parameter:
+  - Force
+  - NumberOfResultsToSkip
+  - CorrelationId
+  - Verb
+  - ResultSize
+- LicenceState 
```

Comment:
can you please fix the indentation?

Reply:
This indentation is intentional, line 35,36 and 37 are sub points while 38 is a point by itself

Reply:
ok